docs/tool-comparison.md

```diff
@@ -0,0 +1,84 @@
+# Comparison of GitOps Promotion Tools
+
+A GitOps user defines their desired state as files in a git repo. To deploy a change, the user modifies a file and 
+pushes a commit. To automatically promote that change, you need a system that knows these things:
+
+1. What to promote
+
+    Some stuff makes sense to promote, and some does not. You generally want to promote image tag upgrades. But you 
+    generally don't want to promote environment-specific config like resource requests/limits. The tool needs a way of 
+    knowing what's promotable and what is not.
+
+2. How to "hold" a change before promotion
+
+    The promotion tool needs somewhere to keep a change while it's waiting to be qualified for the next higher 
+    environment.
+
+3. When to make the change live
+
+    For "promotion" to be useful, there must be rules about when a change proceeds to the next step. The promotion tool 
+    needs a way to determine what qualifying rules apply to which environments.
+
+Different tools have different ways of handling these three concepts. Below we summarize the tools, in the order they 
+were introduced.
+
+## The Options at a Glance
+
+| Tool                       | What to promote                                 | How to "hold" the change                              | When to promote                                     |
+|----------------------------|-------------------------------------------------|-------------------------------------------------------|-----------------------------------------------------|
+| **GitOps Promoter**        | Environment hydrated manifests                  | Automated PR (hydrated branch)                        | When commit status and branch protection rules pass |
+| **Home-Grown CI Solution** | Whatever you want                               | Whatever you want                                     | Whatever you want                                   |
+| **Telefonistka**           | Environment directory contents                  | Automated PR (DRY branch)                             | When branch protection rules pass                   |
+| **Kargo**                  | git commits, image tags, or Helm chart versions | A variety of options (PRs, commits, out-of-sync apps) | Defined by a DAG and a variety of rules             |
+| **Codefresh Products**     | json-path-specified fields                      | Codefresh backend                                     | Manually or according to promotion rules            |
+
+## GitOps Promoter
+
+Where other tools mostly operate on "DRY," pre-rendered manifests, GitOps Promoter doesn't touch those files.
+Instead, the user is expected to make a manifest change to affect all target environments (e.g. an image tag bump in a
+shared Kustomize base), and the Promoter will open PRs to environment-specific "rendered manifest" branches. The
+Promoter renders one branch per environment directory.
```

Comment:
This part `Promoter renders one branch per environment directory.` is slightly confusing. I think it assumes to much Kustomize as a pattern possibly?